### ISSUE TITLE
Add the SFT tutorial 

### ISSUE THREAD

**Issue by shengyangs** [NVIDIA/NeMo-Aligner#59]
**Is your feature request related to a problem? Please describe.**

We should include a tutorial for the SFT. Although we have SteerLM, including a SFT tutorial is important because it is the simplest technique for a user to get started. It is also prerequisite of RLHF and DPO.

**Comment by shengyangs** [comment]
@gshennvm @odelalleau  How do you think?

**Comment by odelalleau** [comment]
> @gshennvm @odelalleau How do you think?

I agree we should have one.

**Comment by gshennvm** [comment]
there is a SFT section in the RLHF tutorial, we can pull that out -- will that do?

**Comment by gshennvm** [comment]
see: https://github.com/NVIDIA/NeMo-Aligner/blob/main/docs/user-guide/RLHF.rst?plain=1#L51

**Comment by gshennvm** [comment]
also @shengyangs do you happen to have a better dataset/script for the SFT tutorial? I wonder if we should update to something other than dolly

**Comment by shengyangs** [comment]
@gshennvm You are right. The SFT is here already. I think we probably want to pull it out in a separate section. I missed it in the first read. I was talking with some people, and they are interested in trying out SFT due to its simplicity.

**Comment by shengyangs** [comment]
The current dolly dataset is fine to me as a prompt-response example. Maybe we should add another example with a chat dataset, for it I have been playing with Ultrachat. I am not sure if there are simpler toy datasets.
